src/IndexingWrapper.cpp

```diff
@@ -243,6 +245,71 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
 	    }
 	}
 
+	//Add value slots
+	for (Rcpp::List::iterator it = slotList.begin(); it != slotList.end(); ++it) {
+	    Rcpp::List list = it.operator*();
+	    bool serialiseVal = false;
+	    bool dfVal = true; // use original row values in data frame
```

Comment:
I'd prefer this to have a clearer variable name. One way would be to use an enum type, so you have two situations: `values_from=DATA_FRAME` and `values_from=LIST_VALUES` (or similar names). Alternatively, you could have a name like `override_frame_values` (with the opposite sense of the variable you have now).